catalog/admin/module/catalog-admin-module-sources/pom.xml

```diff
@@ -151,9 +151,10 @@
             <version>0.1.10</version>
         </dependency>
         <dependency>
-            <groupId>org.webjars.bower</groupId>
-            <artifactId>bootswatch</artifactId>
-            <version>3.3.7</version>
+            <groupId>ddf.distribution.theme</groupId>
+            <artifactId>ddf-theme</artifactId>
+            <version>2.12.0-SNAPSHOT</version>
+            <scope>compile</scope>
```

Comment:
Is there a reason to use `<scope>compile</scope>` explicitly?  This is the default maven scope